scripts/orcid-bulk-push.md

```diff
@@ -0,0 +1,43 @@
+# ORCID-Bulk-Push script
+<!-- TOC -->
+* [ORCID-Bulk-Push script](#orcid-bulk-push-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The ORCID-Bulk-Push script is used to perform a bulk synchronization of all the ORCID entities that are placed in the
+ORCID queue. This script is specifically designed to work with ORCID integration in DSpace.
```

Comment:
I'd recommend linking to the ORCID integration docs on this line: https://wiki.lyrasis.org/display/DSDOC7x/ORCID+Integration